docs/source/conf.py

```diff
@@ -16,6 +16,11 @@
 
 extensions = [
     'myst_nb',
+    'sphinx.ext.duration',
+    'sphinx.ext.doctest',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
```

Comment:
I think lines 20-22 are unnecessary because we don't have any API docs.

Reply:
Removed